plugins/git/gitblame.nix

```diff
@@ -5,67 +5,135 @@
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.gitblame;
+  inherit (lib.nixvim) defaultNullOpts;
```

Comment:
`inherit (lib) types` would also be ok IMO. Or even `types = lib.nixvim.nixvimTypes`?

Off topic: should we consider merging our types into `lib.types` (like we now do for maintainers)? Or is it better to keep our types under the `lib.nixvim` namespace?

Reply:
> `inherit (lib) types` would also be ok IMO. Or even `types = lib.nixvim.nixvimTypes`?
> 
> Off topic: should we consider merging our types into `lib.types` (like we now do for maintainers)? Or is it better to keep our types under the `lib.nixvim` namespace?

Why not mixing them if the two sets don't clash.

Reply:
The only reason I can think of is to be explicit about what is our custom stuff and what is from nixpkg's lib.

How important that is is up for debate. I think my preference would be to merge our types into `lib.types`.

Reply:
See #1978